tools/model-compiler/src/main/java/io/spine/tools/compiler/validation/VBuilderCode.java

```diff
@@ -129,12 +129,13 @@ private File writeClass(String javaPackage, TypeSpec classToCreate) {
                     .writeTo(targetDir);
 
             File createdFile = resolve(javaPackage, classToCreate.name);
-            _debug("The {} class created, written to file {}.", vbClass, createdFile);
+            _debug().log("The `%s` class created, written to file %s.", vbClass, createdFile);
             return createdFile;
 
         } catch (IOException e) {
             String exMessage = format("%s was not written.", targetDir);
-            _warn(exMessage, e);
+            _error().withCause(e)
```

Comment:
I'm not sure it worth logging the cause here while we're immediately re-throwing the exception.